kivy/core/video/video_ffpyplayer.py

```diff
@@ -256,8 +256,6 @@ def _next_frame_run(self):
 
         ev_trigger = self._ev_trigger
         while not self._ffplayer_need_quit:
-            ev_trigger.clear()
```

Comment:
this is actually the correct place for clearing the event if the remaining sleep calls are replaced. but it should be checked against is_set(), because clear() always acquires a lock.

Reply:
I changed the code entirely so this should not be an issue.